papis/commands/doctor.py

```diff
@@ -633,12 +635,85 @@ def biblatex_required_keys_check(doc: papis.document.Document) -> List[Error]:
             if not any(key in doc or aliases.get(key) in doc for key in keys)]
 
 
+BIBLATEX_KEY_RENAME_CHECK_NAME = "biblatex-key-rename"
```

Comment:
I'm really not a fan of this check name. Any suggestions?

I wanted something vaguely general, since this seems like it could be used on other things too, but maybe it's better to make it simple and focused..